sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/Membership.java

```diff
@@ -18,7 +18,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Membership implements Serializable {
     private static final long serialVersionUID = 9064197572478554735L;
-    public static final String OBJECT_TYPE_GROUP = "Group";
+    public static final String OBJECT_TYPE_GROUP = "#microsoft.graph.group";
```

Comment:
Please update `GraphClient`, use `OBJECT_TYPE_GROUP ` instead of `#microsoft.graph.group`.